docs/translations/translations.json

```diff
@@ -213,6 +213,21 @@
     "/styles": "Styles",
     "/styles/basics": "Basics",
     "/styles/advanced": "Advanced",
-    "https://themes.material-ui.com/": "Premium Themes"
+    "https://themes.material-ui.com/": "Premium Themes",
+    "/components/material-icons": "Material Icons",
+    "/components/textarea-autosize": "Textarea Autosize",
+    "/components/rating": "Rating",
+    "/components/skeleton": "Skeleton",
+    "/components/tree-view": "Tree View",
+    "/customization/density": "Density",
+    "/blog": "Blog",
```

Comment:
I don't think that we should translate the blog.

Reply:
We’ll need an exclusion rule in the script to stop them from being included.

Reply:
I agree. I have removed the entries by hand so far.